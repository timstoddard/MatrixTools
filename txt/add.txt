3
3
1,2,3
3,2,1
0,9,8
3
3
2,4,6
-5,7,2
9,0,1
3
3
6,9,2
-6,1,7
2,4,-4