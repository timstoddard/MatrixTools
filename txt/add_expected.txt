3
3
9,15,11
-8,10,10
11,13,5